invenio_theme/assets/semantic-ui/less/invenio_theme/theme/elements/container.overrides

```diff
@@ -32,3 +32,13 @@
     justify-content: center;
   }
 }
+
+.ui.container.dashboard {
```

Comment:
could you add the `.invenio-administration` as a parent of this rule ? lets keep the admin styles contained within one namespace